nibabel/gifti/gifti.py

```diff
@@ -427,14 +425,12 @@ def add_gifti_data_array(self, dataarr):
         """
         if isinstance(dataarr, GiftiDataArray):
             self.darrays.append(dataarr)
-            self.numDA += 1
```

Comment:
Not sure about tests, will check.
This line was removed as I made `numDA` a computed read-only property.